### PR TITLE
Add "Files" section to unicode topic

### PR DESCRIPTION
**Pull request by fdemmer**
Document improvements to help avoiding unicode problems with file uploads.

Relates to two open tickets:
- https://code.djangoproject.com/ticket/17686
- https://code.djangoproject.com/ticket/17816

(@fako helped)

### CODE REVIEW

**Comment by charettes** [comment]
I'll let a native speaker review the exact wording but as someone who struggled with this exact issue in my early Django days this clearly explains what action should be taken.

File: docs/ref/unicode.txt [hunk 1]
```diff
@@ -352,6 +352,33 @@ A couple of tips to remember when writing your own template tags and filters:
   translation objects into strings. It's easier to work solely with Unicode
   strings at that point.
 
+Files
+=====
+
+If you're taking advantage of the internationalization features of Django (see
```
Comment (claudep) [inline docs/ref/unicode.txt]: I think that we could remove the "if" part. Any app accepting uploads should be ready to receive file names with Unicode chars, it's not the developer decision, it's the user's. I would start the sentence by "If you intend to allow...".

File: docs/howto/deployment/wsgi/modwsgi.txt [hunk 2]
```diff
@@ -74,6 +74,17 @@ should put in this file, and what else you can add to it.
...
+
+    If you get a ``UnicodeEncodeError`` when uploading files with file names
+    that contain non-ASCII characters, make sure Apache is configured to accept
+    non-ASCII file names (see :doc:`/ref/unicode`)::
+
+        export LANG='en_US.UTF-8'
+        export LC_ALL='en_US.UTF-8'
+
+    A common location to put this configuration is ``/etc/apache2/envvars``
+
```
Comment (collinanderson) [inline docs/howto/deployment/wsgi/modwsgi.txt]: +1

**Comment by timgraham** [comment]
merged in 25b912abbe31fa440e702b5273c18cf74e2d6e0b, thanks!

**Pull request closed without merging**